glanzb/homework1.md

```diff
@@ -0,0 +1,183 @@
+### Homework #1
+(Due Monday 1/12)
+
+You need to turn in this homework by cloning this repo, making a new branch with your solutions, pushing to github, and issuing a pull request.
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_Difficulty: mixed_)
+Predict the output of each of the following expressions, and explain the rules which determine it.
+If the output depends on the value of x, identify the conditions when the expression will output true.  Assume the cases are independent, and x is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
+
+**a)** `"1" == 1`
+	true, number and string 1 are similar enough, js vill convert the string into number
+
+**b)** `"1" === 1`	->	false, string 1 and number one are not equal
+
+**c)** `x == 'x'` -> error, x is not defined; true if x = 'x'
+
+**d)** `x == (x+'')`	->	error, x is not defined; true if x is a number, false if x is a string	
+
+**e)** `'' == ' '`	-> false, string 2 space is not the same value as string space, differ in length 
+
```

Comment:
e) actually in javascript these mean two very different things. '' <-quotes with no spaces  are a shortcut for NULL which means the absence of information. (an important thing to know for writing functions). ' ' <--- is a string with a space in it.